workers/unity/Packages/io.improbable.gdk.core/Utility/Snapshot.cs

```diff
@@ -50,6 +65,11 @@ public EntityId AddEntity(EntityTemplate entityTemplate)
         /// </remarks>
         public void AddEntity(EntityId entityId, EntityTemplate entityTemplate)
```

Comment:
I can't auto-add persistence here, the `Persistence` struct is code gen and the core package has no access to it. I think if you keep forgetting it, just add an extension method.

Reply:
You can, you have to use the schema APIs though. 

See https://github.com/spatialos/gdk-for-unity/pull/1477/files#diff-6cbb5e5c9ae6005cf38f5951cabcfbecR58 for an example